app/models/tournament.rb

```diff
@@ -21,4 +21,120 @@
 
 class Tournament < Event
   enum game_modes: [:ko, :ko_group, :double_elimination]
+
+  def standing_of(team)
+    get_standing_of team, finale
+  end
+
+  def finale
+    finale = nil
+    matches.each do |match|
+      if match.depth == 0
+        finale = match
+      end
+    end
+    #TODO: keep care of 3rd place match here
+    finale
+  end
+
+  def generate_schedule
+    build_matches filled_teams, max_match_level
+    set_match_indices
+  end
+
+  def max_match_level
+    Math.log(teams.length, 2).ceil - 1
+  end
+
+  private
+
+    def get_standing_of(team, match)
```

Comment:
I am not happy with the naming of this method, because of the second parameter.
Before having a closer look, i wondered how a team could have a standing in a match.
Maybe rename to lastMatchOf(team, currentMatch) and move the standing_string call to standing_of(team) method?